go.mod

```diff
@@ -136,8 +137,10 @@ require (
 	github.com/google/go-cmp v0.6.0 // indirect
 	github.com/google/go-containerregistry v0.14.0 // indirect
 	github.com/google/go-github/v50 v50.1.0 // indirect
+	github.com/google/go-licenses v1.6.0 // indirect
```

Comment:
You need to add this to tools.go, or else it will be removed with `go mod tidy`, which you should run, btw

Reply:
Good to know. I was wondering how it would not get auto-removed. Done.